train.py

```diff
@@ -0,0 +1,164 @@
+import uuid
+import itertools
+from typing import Callable
+
+import hydra
+import torch.optim
+import torch.nn as nn
+from tqdm import tqdm
+from omegaconf import DictConfig
+from torch.utils.data import DataLoader
+
+import wandb
+from model import PitchSeqNN
+from data.dataset import BagOfPitches
+
+
+@hydra.main(version_base=None, config_path="conf", config_name="config")
+def main(cfg: DictConfig):
+    # loading data
+    dataset = BagOfPitches()
+    v_dataset = BagOfPitches(split="validation")
+    train_dataloader = DataLoader(dataset, batch_size=cfg.hyperparameters.batch_size, shuffle=True)
+    v_dataloader = DataLoader(v_dataset, batch_size=cfg.hyperparameters.batch_size)
+
+    # get composers to classify against each other
+    count = dataset.df.groupby(["composer"]).size()
+    composers_with_most_data = count[count.values > 10]
+    composers_to_check = composers_with_most_data.index.tolist()
+    composers = [pair for pair in itertools.combinations(composers_to_check, r=2)]
+    print(f"pairs to check: {len(composers)}")
+
+    # container for trained models
+    models = []
+
+    # train model for each pair
+    for pair in composers:
```

Comment:
Not sure if I'm missing something here, but composer pair is an argument in the `ComposerClassificationDataset`, and you are not passing this argument at any point - it's only used for the experiment name. 

So, it looks like this loop will always run on the default set of Chopin vs Beethoven?

Reply:
Yeah, I realized that, I am running the training again : ))))